addons/mail/tests/test_mail_channel.py

```diff
@@ -359,7 +359,7 @@ def test_channel_user_synchronize(self):
     def test_channel_info_get(self):
         # `channel_get` should return a new channel the first time a partner is given
         initial_channel_info = self.env['mail.channel'].channel_get(partners_to=self.test_partner.ids)
-        self.assertEqual(set(p['id'] for p in initial_channel_info['members']), {self.partner_employee_nomail.id, self.test_partner.id})
+        self.assertEqual(set(m['persona']['partner']['id'] for m in initial_channel_info['channel']['channelMembers'][0][1]), {self.partner_employee_nomail.id, self.test_partner.id})
```

Comment:
Would put a brief comment of expected shape, which would explain the reason for `[0][1]` and help adapt when it doesn't work anymore